### PR TITLE
pandora-wrapper fixes

### PR DESCRIPTION
**Pull request by jonasfoe**
Hey Dario,

I had trouble running my old v1 install so I tried the development version.
To get everything how I needed it I had to make some fixes.
Thanks for the great work and I am looking forward to using a full v2 release soon :)

Best,
Jonas

### CODE REVIEW

File: PANDORA/Pandora/Modelling_functions.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def predict_anchors_netMHCIIpan(peptide, allele_type, output_dir, verbose=True,
     # Retrieves the enviroment variable netMHCIIpan
     netmhcpan_file_path = set([x for x in [os.getenv('netMHCIIpan', default=None), 
                           os.popen('which netMHCIIpan').read().strip()] 
-                          if type(x) == str and re.search("\/netMHCIIpan\-\d+\.\d+\/netMHCIIpan$", x)])
+                          if type(x) == str])
```
Comment (DarioMarzella) [inline PANDORA/Pandora/Modelling_functions.py:134]: Can I ask you why did you remove this strict check? I agree is a bit redundant, but was it causing any specific issue?

File: PANDORA/cmd_pandora.py [hunk 2]
```diff
@@ -226,4 +226,4 @@ def cmd_run_wrapper():
         IDs_col=args.targets_id_column, peptides_col=args.peptides_column,
         allele_name_col=args.allele_name_column, anchors_col=args.anchors_column,
         n_loop_models=args.loop_models, collective_output_dir=args.output_path,
-        clip_C_domain=args.clip_C_domain)
\ No newline at end of file
+        clip_C_domain=args.clip_C_domain, use_netmhcpan=True)
```
Comment (DarioMarzella) [inline PANDORA/cmd_pandora.py:229]: Could I please ask you to set the use_netmhcpan argument to False and add an argument to the parser to turn it to true? Although I agree that in most of the cases a user wants to use netmhcpan, it is only an optional dependency and it would cause this script to crash in case a user does not have it installed.
If it's an issue for you I can also commit directly to your branch and take care of it.

**Review by DarioMarzella — CHANGES_REQUESTED** [review]
@jonasfoe Thanks a lot for the kind message and for the fixes! Could you please check my comments?

Also, our next release will most likely come during this week, as soon as I am done fixing the testing and implementing the last PRs.

File: PANDORA/Pandora/Modelling_functions.py (hunk 1 above)
Comment (jonasfoe) [reply PANDORA/Pandora/Modelling_functions.py:134]: I have a working version of netMHCpan installed on windows in wsl. I have it placed in a folder under a custom name so other users on the workstation don't confuse it.
netMHCpan can be difficult and inflexible to install with the way its implemented using absolute paths. So I was just happy to have it running. But then pandora just threw an error stating it's not installed even though it's perfectly usable when removing the check.
I am not sure what os.getenv('netMHCIIpan') does but if "which netMHCpan" works I am not sure it's wise to add further restrictions. Maybe one do a quick check of netMHCpan --version or something like this.
I mentioned in the commit message: "If this is unavoidable I would request for this requirement to be documented or a precise error to be thrown". Otherwise it's likely to cause a lot of confusion finding out what is the point of failure.

File: PANDORA/cmd_pandora.py (hunk 2 above)
Comment (jonasfoe) [reply PANDORA/cmd_pandora.py:229]: I think it's best if you add that yourself as I unfortunately don't really have a dev environment set up and was rather just adding quick fixes.
The reason I did it like this is that it seems to me that one either provided anchors (-k) via the .tsv, in which case netMHCpan is not tried or without -k there is no alternative to using netMHCpan.
I probably missed some edge case?

File: PANDORA/Pandora/Modelling_functions.py (hunk 1 above)
Comment (DarioMarzella) [reply PANDORA/Pandora/Modelling_functions.py:134]: I see, thanks a lot for the explanation.
The reason why we used both os.getenv and netmhcpan is because, depending on how the user installed netmhcpan (i.e. by setting an alias or by adding it to the PATH) they return different results, so we wanted to make sure to catch them all.

File: PANDORA/cmd_pandora.py (hunk 2 above)
Comment (DarioMarzella) [reply PANDORA/cmd_pandora.py:229]: Sure, I can easily do it.

A user might also want (for MHC-I only) to simply model the cases by using canonical anchoring, although I admit it's not the best, as it would always be better to predict the anchors somehow. 
I will make sure I also add some warnings about it.

**Comment by DarioMarzella** [comment]
Moved to #238
